app/views/jobs/index.html.erb

```diff
@@ -3,5 +3,9 @@
 <div class="flash-container"></div>
 <%= render '/partials/sidebar' %>
 <div class="jobs">
-  <%= render partial: "/partials/jobs", locals: { partial: "jobs_buttons" } %>
+  <% if @jobs.empty? %>
```

Comment:
This could be a good place to use presenters, although it's probably a little late in the game to include them. We have to leave something for 1412 to do :smirk_cat: